app/gui/src/project-view/components/visualizations/ScatterplotVisualization.vue

```diff
@@ -861,7 +873,10 @@ config.setToolbar([
 </script>
 
 <template>
-  <div ref="containerNode" class="ScatterplotVisualization">
+  <div v-if="errorMessage && errorMessage.length > 0" class="WarningsScatterplotVisualization">
```

Comment:
nit: i prefer `errorMessage !== ''`

Reply:
I have changed this so when there is no error message it is just set to `null` rather than an empty string so the check can just `if="errorMessage`